rosie/src/main/java/com/karumi/rosie/view/RosieFragment.java

```diff
@@ -59,9 +59,7 @@
       Bundle savedInstanceState) {
     injectDependencies();
     int layoutId = getLayoutId();
-    View view = inflater.inflate(layoutId, container, false);
-    ButterKnife.bind(this, view);
-    return view;
+    return inflater.inflate(layoutId, container, false);
```

Comment:
To be consistent I'd add an `onPrepareFragment` method to let clients bind their views from their fragments as well. Include the view as a parameter.